### PR TITLE
Publish GA release post

### PR DESCRIPTION
**Pull request by dmuelle**
post on staging: https://blogs-staging-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2024/07/16/24.0.0.7.html

### CODE REVIEW

**Comment by dmuelle** [comment]
This PR also includes previously approved Japanese translation of 24.0.0.5 and a 3rd party post

File: posts/2024-07-11-virtual-threads-case-study.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+redirect_link: https://www.infoq.com/articles/java-virtual-threads-a-case-study/
+permalink: /blog/redirected.html
+---
+= Java Virtual Threads: A case study
+Gary DeVal <https://github.com/gjdeval>
+//Blank line here is necessary before starting the body of the post.
+
+You can run your own apps using Java Virtual Threads on Open Liberty but, for now, we won’t be replacing Open Liberty’s existing self-sizing thread pool with virtual threads.
+
+InfoQ.com have today published our article about what our performance team found when they compared Java Virtual Threads with Liberty’s autonomic thread pool. Including some bugs which are still being investigated with the OpenJDK community.
```
Comment (ayoho) [inline posts/2024-07-11-virtual-threads-case-study.adoc]:
```suggestion
InfoQ.com have today published our article about what our performance team found when they compared Java Virtual Threads with Liberty’s autonomic thread pool, including some bugs which are still being investigated with the OpenJDK community.
```

File: posts/2024-07-11-virtual-threads-case-study.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Java Virtual Threads - A case study"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/gjdeval
+author_github: https://github.com/gjdeval
+seo-title: Java Virtual Threads - A case study - OpenLiberty.io
+seo-description: Run your own apps using Java Virtual Threads on Open Liberty but, for now, we won’t be replacing Open Liberty’s existing self-sizing thread pool with virtual threads. InfoQ.com have today published our article about what our performance team found when they compared Java Virtual Threads with Liberty’s autonomic thread pool. Including some bugs which are still being investigated with the OpenJDK community.
```
Comment (ayoho) [inline posts/2024-07-11-virtual-threads-case-study.adoc]:
```suggestion
seo-description: Run your own apps using Java Virtual Threads on Open Liberty but, for now, we won’t be replacing Open Liberty’s existing self-sizing thread pool with virtual threads. InfoQ.com have today published our article about what our performance team found when they compared Java Virtual Threads with Liberty’s autonomic thread pool, including some bugs which are still being investigated with the OpenJDK community.
```

File: posts/2024-07-11-virtual-threads-case-study.adoc [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+blog_description: "You can run your own apps using Java Virtual Threads on Open Liberty but, for now, we won’t be replacing Open Liberty’s existing self-sizing thread pool with virtual threads. InfoQ.com have today published our article about what our performance team found when they compared Java Virtual Threads with Liberty’s autonomic thread pool. Including some bugs which are still being investigated with the OpenJDK community."
```
Comment (ayoho) [inline posts/2024-07-11-virtual-threads-case-study.adoc]:
```suggestion
blog_description: "You can run your own apps using Java Virtual Threads on Open Liberty but, for now, we won’t be replacing Open Liberty’s existing self-sizing thread pool with virtual threads. InfoQ.com have today published our article about what our performance team found when they compared Java Virtual Threads with Liberty’s autonomic thread pool, including some bugs which are still being investigated with the OpenJDK community."
```

File: posts/2024-07-11-virtual-threads-case-study.adoc (hunk 1 above)
Comment (dmuelle) [reply posts/2024-07-11-virtual-threads-case-study.adoc]: Resolved in #3875

**Review by ayoho — APPROVED** [review]

**Pull request merged**